docker/graph.Dockerfile

```diff
@@ -0,0 +1,6 @@
+# In order to support multiple development environments we use a custom base
+# node built from the graph node repo
+# See: https://github.com/graphprotocol/graph-node
+FROM ghcr.io/hypercerts-org/graph-node:3d04464ccfe08f3850b58dfcf9c6afc12f726a44
```

Comment:
This is probably where my Docker build broke? How important is it that we use our own builds?

Reply:
This is fixed! Should be public now :)